specs/lib/gltfPipelineSpec.js

```diff
@@ -107,19 +107,43 @@ describe('gltfPipeline', function() {
             createDirectory : false
         };
         readGltf(gltfPath, options, function(gltf) {
+            var options = { basePath : path.dirname(gltfPath) };
             processJSONToDisk(gltf, outputPath, options, function() {
-                expect(path.normalize(spy.calls.first().args[0])).toEqual(path.normalize('./output/'));
+                expect(path.normalize(spy.calls.first().args[0])).toEqual(path.normalize('output/output'));
+                done();
+            });
+        });
+    });
+
+    it('will write sources from JSON', function(done) {
+        var options = {};
+        readGltf(gltfPath, options, function (gltf) {
+            var initialUri = gltf['buffers'].CesiumTexturedBoxTest.uri;
+            processJSON(gltf, options, function () {
+                var finalUri = gltf['buffers'].CesiumTexturedBoxTest.uri;
+                expect(initialUri).not.toEqual(finalUri);
```

Comment:
The initial uri will be a filepath to the .bin file, so the comparison will always pass. A quick fix for these 2 tests is to read from `gltfEmbeddedPath` instead of `gltfPath`.